Tools/autotest/sim_vehicle.py

```diff
@@ -21,12 +21,14 @@
 import textwrap
 import time
 import shlex
+from geographiclib.geodesic import Geodesic
```

Comment:
please use:
from MAVProxy.modules.lib import mp_util
and use gps_newpos(). That removes a dependency on geographiclib